course_info/icommons.py

```diff
@@ -59,6 +70,11 @@ def _get_resource(self, type_, id_, **kwargs):
         path = '{}/'.format(type_)
         if id_:
             path += '{}/'.format(id_)
+
+        # check for 'append_to_path' kwarg and append that value to the resource url
+        if kwargs.get("append_to_path"):
```

Comment:
This method is hopefully operating at the level of a restful resource...it shouldn't be expecting parameters that care about url paths.  I think we can keep the code as-is, and just rename the parameter to `sub_resource` or something along those lines, since we're really asking for a child of the `course_instances` resource, not the `course_instance` itself.

Another approach would be to just look for `type='course_instance_people`, and construct the url accordingly.